openedx/features/redhouse_panel/api/v0/users_account_stats.py

```diff
@@ -0,0 +1,82 @@
+from django.contrib.auth import get_user_model
```

Comment:
Please rename this file to `Views.py`. We are not going to create separate files for each view.

Reply:
@imhassantariq In order to keep consistency with the project structure, I have moved the code in `views.py`. But don't you think it's a good approach to have multiple files instead of a single `views.py` when you have multiple APIs which are serving a different purpose i.e analytics / CRUD APIs.

Reply:
We can do that indeed, but right now, as we are building the API from scratch and at this stage, it's good to not complicate things for now.